### PR TITLE
add(component-page): Content group banner

### PR DESCRIPTION
**Pull request by shixiedesign**
### Related Ticket(s)

https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/issues/891

### Changelog

**New**

- adds `Content group — banner` page along with a image for it

### CODE REVIEW

**Comment by JennySanchez** [comment]
@shixiedesign Based on our conversation, we need to add the "under construction" to the React status column under the "Status" for this PR. 

We also might need to update the sentence for the "Design and functional specifications" section based on the [Slack conversation](
https://cognitive-app.slack.com/archives/G01CFF8ETPU/p1618858948020000).

**Comment by shixiedesign** [comment]
@JennySanchez Should be added now: https://ibmdotcom-library.s3-web.us-south.cloud-object-storage.appdomain.cloud/deploy-previews/940/components/content-group-banner

**Comment by JennySanchez** [comment]
@shixiedesign Updated the sentence based on the Slack discussion on Slack. Other than that, we should be good to go.

File: src/pages/components/content-group-banner.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: Content group – banner
+description: Content group &mdash banner is designed as a subtle callout, intended for tangential information or announcements supplementary to the page's main focus.
```
Comment (oliviaflory) [inline src/pages/components/content-group-banner.mdx:3]: should there be a comma after "announcements" ? 

Content group &mdash banner is designed as a subtle callout, intended for tangential information or announcements, supplementary to the page's main focus.

or

Content group &mdash banner is designed as a subtle callout, intended for tangential information or announcements that are supplementary to the page's main focus.

File: src/data/components.json [hunk 2]
```diff
@@ -886,6 +886,32 @@
         "notAvailable": false
       }
     },
+    "Content group — banner": {
+      "url": "./content-group-banner",
+      "description": "Content group — banner is designed as a subtle callout, intended for tangential information or announcements that are supplementary to the page's main focus.",
+      "storybook": {
+        "webcomponents": "http://ibmdotcom-web-components-canary.mybluemix.net/?path=/story/components-content-group-banner--default",
```
Comment (oliviaflory) [inline src/data/components.json:893]: I may be mistaken here, but I didn't think we usually link to Canary storybooks? We should link the the stable one when the release goes out? @JennySanchez cc @shixiedesign

**Review by oliviaflory — APPROVED** [review]
Looks good to me, release is live so we are good to go!

**Comment by JennySanchez** [comment]
FYI - I found some typos and fix them along with the "above the fold" edit, and we should be good to go now.
Screen Shot 2021-04-22 at 8 37 54 AM

**Review by JennySanchez — COMMENTED** [review]
We are good to go. Thanks!

**Pull request merged**